templates/serviceworker.js

```diff
@@ -0,0 +1,45 @@
+const cacheName = 'staticFiles';
+
+// during the install phase you usually want to cache static assets
+self.addEventListener('install', (e) => {
+  // once the SW is installed, go ahead and fetch the resources to make this work offline
+  e.waitUntil(
+    caches.open(cacheName)
+      .then((cache) => {
+        return cache.addAll([
+          '/',
+          '/offline'
+        ])
+          .then(() => {
+            self.skipWaiting();
+          })
+          .catch(e => console.error(e));
+      })
+      .catch(e => console.error(e))
+  );
+});
+
+// when the browser fetches a url
+self.addEventListener('fetch', (event) => {
+  if (event.request.url.match(/http:\/\/.*((\.(css|js|svg)$)|(placeholder\d+x\d+\.png$))/)) {
+    console.log(event.request.url);
```

Comment:
Please remove this and other occurrences of console.log and any other debug code.